### ISSUE TITLE
2K12M3 Kub-M3 - The good SA-6

### ISSUE THREAD

**Issue by FrangibleCover** [PygmalionOfCyprus/cmo-db-requests#4250]
### DB Selector

Both

### DBID to Copy

Facility 437 (CWDB), Facility 132 (DB3k)

### Hypothetical

No

### Name

SAM Bty (SA-6a Gainful) [2K12M3 Kub-M3])

### Country

Soviet Union

### Service

Army

### In Commission

1976-1991

### Sensors

as copy

### Mounts

Replace all missiles with CWDB 893/DB3k 1824 - 3M9M3 missiles, with #4253 modifications. These mounts are called 2P25M2.

In principle the battery could fire base 3M9 and 3M9M1 missiles as well. Post-breakup Czech training seems to have been done entirely with 3M9M1s, with the M3s saved up as warshots.

### Other Changes

I know these things were operated by a good number of foreign users of the Kub/Kvadrat system, and I've worked out how to get visual proof: Cranked delta fins with a thick base means 3M9M3. As a result the sources are going to be full of images and tangientally related articles.

- [ ] Copy for DDR - 1979 entry to service
- [ ] Copy for DDR in CWDB too
- [ ] Copy for Czechoslovakia - 1985 entry to service
- [ ] Replace DBID 4036 for Czechia with this version.
- [ ] Copy for Slovakia - 1993
- [ ] Copy for Poland
- [ ] Copy for Yugoslavia - Unknown entry to service
- [ ] Copy for Serbia - Possible that all vehicles were upgraded and therefore should replace DBID 2572
- [ ] Copy for Hungary
- [ ] Change brand new Hungarian SA-6 upgrade (#4168) to use 3M9M3 on 2P25M2 mounts - Sorry!
- [ ] Copy for India
- [ ] Copy for Iraq
- [ ] Replace DBID 4430 for Romania with this version.
- [ ] Version for Iran. Fater-1 looks like a 3M9M3. If this is the case then the Iranians almost certainly only got M3 variants.
- [?] Copy for all post-Soviet SA-6 operators? 
- [ ] Copy for Russia
- [ ] Copy for Armenia
- [ ] Copy for Turkmenistan - honestly if Turkmenistan have them I'd be surprised if any Post-Soviet state doesn't.
- [ ] Copy for Ukraine
- [ ] Maybe change DBID 3925 to this config or maybe change it to the #4251 config

I'm pretty sure this isn't a full list. The Soviets do not appear to have been shy about selling the system, especially once the Buk had entered service and the M3 was old-hat.

### Sources

https://odin.tradoc.army.mil/WEG/Asset/2K12_Kub_(SA-6_Gainful)_Russian_Surface-to-Air_Missile_System
https://en.wikipedia.org/wiki/2K12_Kub
Janes
_SA-6 Gainful In Detail_ - F. Koran, J. Horak, T. Bouchal
http://www.flak11.de/2K12-Start.htm
https://gagadget.com/en/226215-not-just-mig-29-fighters-slovakia-to-give-ukraine-2k12-kub-surface-to-air-missile-systems/
https://twitter.com/balt_security/status/1140941320695009280
![image](https://github.com/PygmalionOfCyprus/cmo-db-requests/assets/109925704/eba156e5-21f9-4726-b38b-7eef75c1e9b4)
https://commons.wikimedia.org/wiki/Category:2P25_TELs_in_Romanian_service#/media/File:Romanian_2K12_Kub_(SA-6)_6.jpg
https://commons.wikimedia.org/wiki/Category:2P25#/media/File:War_trophies_from_Armenian_Army_at_2020_Victory_Parade_in_Baku_6.jpg
https://commons.wikimedia.org/wiki/Category:2P25_TELs_in_Iraqi_service#/media/File:SA-6Gainful.jpg
https://commons.wikimedia.org/wiki/Category:2P25_TELs_in_Polish_service#/media/File:KUB_MSPO2003_PICT0023.JPG
https://commons.wikimedia.org/wiki/Category:2P25#/media/File:Independence_Day_Parade_-_Flickr_-_Kerri-Jo_(73).jpg
https://commons.wikimedia.org/wiki/Category:2K12_SAMs_in_Iranian_service#/media/File:2013_Sacred_Defense_exhibition_in_Tehran_(23).jpg - This is something weird because of the radar vehicle but it's certainly 2K12M3 derived so they must have had some.
![image](https://hips.hearstapps.com/hmg-prod/images/ukrainian-2k12-kub-2-64ee45235f516.jpg?resize=980:*)

**Comment by PavelNovak79** [comment]
I do not agree with datalink. See my note here: https://github.com/PygmalionOfCyprus/cmo-db-requests/issues/4253

Czechoslovakia probably obtained its first 3M9M3(E) missiles in 1984 or 1985 and most of Czechoslovak Kub batteries (if not all) were modernized to 2K12M3 standard by the end of 1980s.

2K12M3 can also use older missiles (in CS case 3M9M(E) - we did not have basic 3M9) and even on one launch vehicle can be combination of different variants of missiles.

**Comment by PavelNovak79** [comment]
Czech Republic and Slovak should have start date 1993.
